app/views/categories/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="devise-head">
```

Comment:
![mobile](https://github.com/Munyabelden/Budget-App/assets/27709832/91595041-ba7c-4c5a-b1ee-f3276ba34467)
- [ ] **Optional:** When a User is signed in, there isn't a way to sign out.

Reply:
Thank for the comment mate, noted!!